docker-compose.yml

```diff
@@ -45,27 +45,40 @@ services:
       - ./appeals-service-api:/opt/app
     command: npm run start:dev
 
-  document-service-api:
+  document-service-api: &document-service
     image: node:14-alpine
     environment:
       BLOB_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://blob-storage:10000/devstoreaccount1;QueueEndpoint=http://blob-storage:10001/devstoreaccount1;
+      STORAGE_CONTAINER_NAME: document-service-uploads
+      FILE_MAX_SIZE_IN_BYTES: 50000000
```

Comment:
Have I made a balls up then - https://github.com/foundry4/appeal-planning-decision/blob/master/docker-compose.yml#L75

think I need to change this to `50000000` also ?

Reply:
Yes, and no. In dev, we don't care what the actual size should be. It's a development experience and has no bearing on what it is on the deployed application (and these could conceivably be different in each each environment.

The number you've done there is actually 50 mebibytes - data was decimalised about 10 years ago so a megabyte is 1000^2, but a mebibyte is 1024^2 https://en.wikipedia.org/wiki/Mebibyte

Reply:
I've just read that article. I'm now upset that 1998 wasn't "about 10 years ago".

[I'm old](https://giphy.com/gifs/jamie-lee-curtis-boho-freaky-friday-TfseOfhwd6BJC)